Makefile

```diff
@@ -133,24 +146,26 @@ release: goreleaser ## Runs goreleaser for the operator-controller. By default,
 	$(GORELEASER) $(GORELEASER_ARGS)
 
 quickstart: export MANIFEST="https://github.com/operator-framework/operator-controller/releases/download/$(VERSION)/operator-controller.yaml"
-quickstart: kustomize generate ## Generate the installation release manifests and scripts
+quickstart: kustomize ### Generate the installation release manifests and scripts with dependencies
```

Comment:
Curious why we're removing `generate` as a dependency. Without it, the generated `operator-controller.yaml` may be out-dated.

Reply:
Nevermind, I see `generate` only generates go code. But perhaps `manifests` should be a dependency of `quickstart`?